### ISSUE TITLE
Style guide audit

### ISSUE THREAD

**Issue by bethpottsVADEPO** [department-of-veterans-affairs/va.gov-team#40230]
## Issue Description
We want to make it easier for all VA content authors to use the VA.gov style guide and the AP stylebook, and know which to check first. So we want to identify these points:
- what entries repeat/match guidance in AP (basically, find the dups)
- what entries are exceptions to AP
- what entries are VA-specific

If most of the entries are in one of these categories, there's no need to list them all out. If we have shorter lists, for instance, of dups and exceptions and all others are VA-specific, just list the dups and exceptions and indicate that all other entries are VA-specific. 

**Note:** Once this is complete, Beth will follow up with the Web Governance Board with the results, specifically alerting Lisa Hollenbeck. She raised the issue because she wasn't sure how to know what exceptions there are to AP. We may want to identify those entries that are exceptions--need to think about the content author user experience and what will be most helpful. 

---
## Tasks 
- [ ] What entries repeat/match guidance in AP (basically, find the dups)?
- [ ] What entries are exceptions to AP?
- [ ] What entries are VA-specific?
- [ ] Identify any other categorizations not captured by the above 3.

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by megzehn** [comment]
Attaching as a Microsoft Word doc! [https://app.zenhub.com/files/133843125/fe1898b6-a225-4a61-b38a-4e7bd88f6d2a/download](https://app.zenhub.com/files/133843125/fe1898b6-a225-4a61-b38a-4e7bd88f6d2a/download)

**Comment by RLHecht** [comment]
Thanks, @megzehn this looks great. @bethpottsVADEPO can you take a look at the word doc and I can create a new ticket for any updates we would want to make to the style guide based off of this?

**Comment by bethpottsVADEPO** [comment]
@megzehn @RLHecht Thanks for this work! I'll take a look next week and we'll figure out next steps.

**Comment by RLHecht** [comment]
Moving this to the icebox until we have time to pick this up again.

**Comment by mistymg** [comment]
Ticket was moved to Closed column but not actually closed.
